package.json

```diff
@@ -2,23 +2,37 @@
   "name": "the8thavenue-api",
   "version": "1.0.0",
   "description": "",
-  "main": "src/server.ts",
-  "scripts": {
-    "start": "NODE_ENV=production node dist/server.js",
-    "dev": "tsc && nodemon src/server.ts",
-    "build": "tsc",
-    "init-env": "printenv > .env"
+  "homepage": "https://github.com/codebynao/the8thavenue-api#readme",
+  "bugs": {
+    "url": "https://github.com/codebynao/the8thavenue-api/issues"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/codebynao/the8thavenue-api.git"
   },
-  "author": "",
   "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/codebynao/the8thavenue-api/issues"
+  "author": "",
+  "main": "src/server.ts",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc && nodemon src/server.ts",
+    "format": "npm run lint -- --fix",
+    "init-env": "printenv > .env",
+    "lint": "eslint --cache --ext .js,.ts .",
```

Comment:
Just a question: we don't need prettier at all, right?

Reply:
Yep